lib/danger/danger_core/plugins/dangerfile_bitbucket_server_plugin.rb

```diff
@@ -150,7 +150,9 @@ def head_commit
     # @return [String]
     #
     def html_link(paths, full_path: true)
-      create_link(paths, full_path) { |href, text| create_html_link(href, text) }
+      # TODO: Enable html formatting when BitBucketServer could handle it
+      #create_link(paths, full_path) { |href, text| create_html_link(href, text) }
+      markdown_link(paths, full_path: full_path)
```

Comment:
@JuanitoFatas, offer a best way to skip these tests:
* using metadata `, skip: true`
* prefixing with `x` like `it -> xit`
* removing tests completely. This way we also do't see any "No reason given" warnings